flexget/plugins/input/anidb_feed.py

```diff
@@ -0,0 +1,479 @@
+            log.error('Unable to parse the XML from url: %s error: %s' % (url, ex))
+            return []
+
+        if not len(parsed_xml.entries) > 0:
+            log.info('No entries returned from xml.')
+            return []
+
+        entries = self.parse_from_xml(parsed_xml.entries, config)
+        if len(entries) == 0:
+            log.verbose('No entries parsed from xml.')
+        # else:
+        #   dump_entry(entries[0])
+
+        return entries
+
+    def make_feedparser_friendly(self, data):
+        try:
+            dom = minidom.parseString(data)
+            entries = dom.getElementsByTagName('entry')
+            for entry in entries:
+                items_ns = entry.getElementsByTagNameNS(NAMESPACE_URL, NAMESPACE_TAGNAME)
+                if items_ns:
+                    for node_ns in items_ns:
+                        dt_nodes = node_ns.getElementsByTagName(NAMESPACE_NAME + ':dt')
+                        dd_nodes = node_ns.getElementsByTagName(NAMESPACE_NAME + ':dd')
+                        if len(dt_nodes) != len(dd_nodes):
+                            logging.warning('unexpected nodecount')
+                            continue
+                        for idx, dt_node in enumerate(dt_nodes):
+                            if len(dt_node.childNodes) >= 1 and len(dd_nodes[idx].childNodes) >= 1:
+                                if dt_node.childNodes[0].nodeValue is not None and dd_nodes[idx].childNodes[0].nodeValue is not None:
+                                    dt_node.tagName = NAMESPACE_NAME + ':%s' % dt_node.childNodes[0].nodeValue.replace(' ', '_')
+                                    dt_node.setAttribute('name', dt_node.childNodes[0].nodeValue)
+                                    dt_node.setAttribute('value', dd_nodes[idx].childNodes[0].nodeValue)
+                                    dt_node.removeChild(dt_node.childNodes[0])
+                                    node_ns.removeChild(dd_nodes[idx])
+                                    entry.appendChild(dt_node)
+        except Exception as ex:
+            logging.debug('Unable to rename nodes in XML: %s' % ex)
+            return None
+        return dom.toxml()
+
+    def set_ns_attribute(self, name, xml_entry, entry, in_ns_name, out_prefix_name=None, in_type=basestring):
+        tagname = in_ns_name + '_' + name  # feedparser represents those as nsname_tagname
+        value = None
+        if tagname in entry and entry.get(tagname) is not None:
+            value = entry.get(tagname)  # use existing, but do type check
+        elif tagname in xml_entry:
+            if isinstance(xml_entry[tagname], dict) and 'value' in xml_entry[tagname]:
+                value = xml_entry[tagname]['value']
+            elif isinstance(xml_entry[tagname], basestring):
+                value = xml_entry[tagname]
+
+        if out_prefix_name is not None and not out_prefix_name.isspace():
+            tagname = out_prefix_name + name
+
+        if value is not None:
```

Comment:
Readability would benefit of reverting the if check to "if value is None" -> log, return so you don't need to indent whole block. Extracting into own method also worthwhile to consider..